### PR TITLE
UI/improvements

### PR DESCRIPTION
**Pull request by arl**
Improvements in the UI (javascript) code

### CODE REVIEW

File: _example/work.go [hunk 1]
```diff
@@ -1,24 +1,44 @@
...
 	"time"
 )
 
 // Work loops forever, generating a bunch of allocations of various sizes in
 // order to force the garbage collector to work.
 func Work() {
-	m := map[string][]byte{}
+	m := make(map[int]interface{})
+	i := 0
+	for ; ; i++ {
```
Comment (davecheney) [inline _example/work.go:13]: Perhaps this could be written as

for i:= 0; ; i++ {

File: statsviz.go [hunk 2]
```diff
@@ -17,17 +18,14 @@ func sendStats(conn *websocket.Conn, frequency time.Duration) error {
...
-		err   error
-	)
+	stats := stats{GoVersion: runtime.Version()}
 	for range tick.C {
 		runtime.ReadMemStats(&stats.Mem)
 		stats.NumGoroutine = runtime.NumGoroutine()
-		if err = conn.WriteJSON(stats); err != nil {
-			break
+		if err := conn.WriteJSON(stats); err != nil {
+			return err
```
Comment (davecheney) [inline statsviz.go:26]: What about break here, then you don’t need the panic “unreachable”

**Review by davecheney — CHANGES_REQUESTED** [review]
Sole drive by comments. Thanks for statsvis, I’m using it on a project at work abd it’s 😘

**Comment by arl** [comment]
> Some drive by comments. Thanks for statsvis, I’m using it on a project at work and it’s 😘

Thanks for the suggestions @davecheney
Glad you found it useful, and even more so since it is loosely inspired by gcvis. 🥰

File: .github/workflows/tests-linux.yml [hunk 3]
```diff
@@ -4,7 +4,7 @@ jobs:
   test:
     strategy:
       matrix:
-        go-version: [1.15.x, 1.16.x]
```
[the remark this answers is not in the document]
Comment (arl) [reply .github/workflows/tests-linux.yml:7]: Good idea

File: _example/work.go (hunk 1 above)
Comment (arl) [reply _example/work.go:13]: Indeed

**Comment by davecheney** [comment]
> > Some drive by comments. Thanks for statsvis, I’m using it on a project at work and it’s 😘 
> Glad you found it useful, and even more so since it is loosely inspired by gcvis. 🥰

mi casa, su casa :)

**Pull request merged**